src/SFA.DAS.ApplyService.Web/Views/Application/Pages/Index.cshtml

```diff
@@ -95,106 +96,111 @@
                                                 @Html.Raw(Model.PageContext)
                                             </span>
                                         }
+
                                         @if (hasSectionTitle)
                                         {
-                                            <span class="govuk-caption-l">@Model.SectionTitle</span>
+                                            <span class="govuk-caption-xl">@Model.SectionTitle</span>
                                         }
+
                                         <h2 class="govuk-label-wrapper">
-                                            <label class="govuk-label govuk-label--l" for="@question.QuestionId">
+                                            <label class="govuk-label govuk-label--xl" for="@question.QuestionId">
                                                 @Html.Raw(question.Label)
                                             </label>
                                         </h2>
-                                        }
+                                    }
 
-                                        @if (question.QuestionBodyText != null)
-                                        {
-                                            @Html.Raw(question.QuestionBodyText)
-                                        }
+                                    @if (question.QuestionBodyText != null)
+                                    {
+                                        @Html.Raw(question.QuestionBodyText)
+                                    }
 
-                                        @if (question.Hint != null)
-                                        {
-                                            <span class="govuk-hint">
-                                                @Html.Raw(question.Hint)
-                                            </span>
-                                        }
+                                    @if (question.Hint != null)
+                                    {
+                                        <span class="govuk-hint">
+                                            @Html.Raw(question.Hint)
+                                        </span>
+                                    }
 
-                                        @if ((question.Type == "ComplexRadio" || question.Type == "ComplexCheckBoxList") && (question.ErrorMessages == null || question.ErrorMessages.Count == 0))
-                                        {
-                                            question.ErrorMessages = Model.ErrorMessages;
-                                        }
-                                        @{
-                                            var questionType = question.Type.Replace("DataFed_", "");
+                                    @if ((question.Type == "ComplexRadio" || question.Type == "ComplexCheckBoxList") && (question.ErrorMessages == null || question.ErrorMessages.Count == 0))
```

Comment:
Same comment about enum for QuestionTypes

Reply:
Raised Jira ticket: https://skillsfundingagency.atlassian.net/browse/APR-1565